tslint.json

```diff
@@ -4,6 +4,22 @@
       "tslint-config-airbnb"
   ],
   "jsRules": {},
-  "rules": {},
+  "rules": {
```

Comment:
Q: Should we update the default configs in the `crank cog:scaffold` output?

Reply:
Yes i think we should since some lint errors are annoying/unecessary. Though some are from my preference so it should be discussed what rules should we exclude